packages/client/src/views/RegisterForm/RegisterForm.tsx

```diff
@@ -271,6 +272,33 @@ class RegisterFormView extends React.Component<FullProps, State> {
         hash: newHash + '-form-input'
       })
     }
+    const oldIsWritingDraft = prevProps.isWritingDraft
+    const newIsWritingDraft = this.props.isWritingDraft
+    if (
+      oldIsWritingDraft !== newIsWritingDraft &&
+      newIsWritingDraft === false
```

Comment:
I think the following logic is also valid `oldIsWritingDraft && !newIsWritingDraft`

Reply:
> I think the following logic is also valid `oldIsWritingDraft && !newIsWritingDraft`. What do you think @soumita-dsi ?

Changed it, vaia.